_updates/2020-04-08-april.md

```diff
@@ -0,0 +1,40 @@
+---
+title: April 2020
+---
+
+## Work From Home 🏠
+
+Because of the pandemic we are now working from home. This doesn't mean that the developement stopped. Here are the changes we made.
+
+### @sensenet/redux@5.3.0
+
+- Added some missing actions for
+  - put
+  - content creation with content template
+  - preview related actions
+  - preview comments
+  - allowedchildtypes
+  - restore from trash
+  - add/remove list fields
+
+### @sensenet/document-viewer-react@2.0.6
+
+- fix: adding comments returned error always, because the x and y coordinates were floating numbers and the request body was not a valid json. These two params' type is changed to string. [#617](https://github.com/SenseNet/sn-client/pull/617)
```

Comment:
```suggestion
- fix: adding comments returned error, because the x and y coordinates were floating numbers and the request body was not a valid json. These two params' type is changed to string. [#617](https://github.com/SenseNet/sn-client/pull/617)
```